src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let countCalls = 0;
+
+  return (a) => {
+    countCalls++;
+
+    if ((countCalls - 3) % 2 === 1) {
```

Comment:
Use countCalls > 3 instead of countCalls - 3, to check if it's after the third call.